api/queues.js

```diff
@@ -49,7 +53,7 @@ router.post(
     ]),
     failIfErrors,
   ],
-  (req, res, _next) => {
+  (req, res, next) => {
```

Comment:
is there a reason this was changed from _next to next?

Reply:
The underscore was used to indicate an unused param, but it's actually used now.